test/route.js

```diff
@@ -418,3 +420,32 @@ test('/documentation2/json route (overwrite)', t => {
       })
   })
 })
+
+test('/documentation/:myfile should return 404 in dynamic mode', t => {
+  t.plan(2)
+  const fastify = Fastify()
+  fastify.register(fastifySwagger, swaggerInfo)
+
+  fastify.inject({
+    method: 'GET',
+    url: '/documentation/swagger-ui.js'
+  }, (err, res) => {
+    t.error(err)
+    t.strictEqual(res.statusCode, 404)
+  })
+})
```

Comment:
Can you please add a test that checks that the Fastify not found handler is called? This is likely to fail/conflict on Fastify v2, and we need the test to make sure the behavior is consistent.

Reply:
Done